### ISSUE TITLE
Seem to have lost max and peak range

### ISSUE THREAD

**Issue by gtjoseph** [wiedehopf/graphs1090#83]
Not sure when this happened but I just noticed that I'm not seeing max or peak ranges no matter how far back in time I look.

![image](https://github.com/wiedehopf/graphs1090/assets/1205666/fbc73240-37e3-479a-b0f7-ecbfd9527e3d)

The lat/lon are set correctly and I can see the max distance in view1090 and in stats.json.  What else should I check?

**Comment by wiedehopf** [comment]
Data corruption on disk?

You can restore a backup from last week.

**Comment by gtjoseph** [comment]
Issue was actually caused by changes I made to stats.json back when I was doing the hirate demod.

**Comment by wiedehopf** [comment]
Pretty sure i recently removed the limit on max range ... i suppose i should have kept something like 20000 nmi in there.

Bit confused why it wouldn't plot the existing values.
I suppose if you have an error message from graphs1090.sh i'd still take a look at that.

**Comment by gtjoseph** [comment]
There weren't any error messages from graphs1090 and the changes I made to dump1090/net_io.c only added new items to the json which your dump1090.py collected plugin should have handled just fine.  It's quite possible though that I inadvertently did make a breaking change.  Reverting my changes fixed the issue and the stats I added weren't all that useful anyway so I probably won't take it any any further.  I'll take a look at the rrd sripts just in case though.  I haven't looked at rrd in depth since wrestling with MRTG a few decades ago so I'm not anxious to go down that rabbit hole. :)
